src/client/src/orangehrmCorporateDirectoryPlugin/components/SummaryCard.vue

```diff
@@ -0,0 +1,306 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <oxd-sheet
+    :gutters="false"
+    class="orangehrm-directory-card"
+    type="white"
+    @click="toggle = !toggle"
+  >
+    <div class="orangehrm-directory-card-header">
+      <oxd-text type="card-title">
+        {{ employeeName }}
+      </oxd-text>
+    </div>
+    <profile-picture :id="id"> </profile-picture>
+    <div class="orangehrm-directory-card-header">
+      <oxd-text type="toast-title">
+        {{ employeeDesignation }}
+      </oxd-text>
+    </div>
+    <div class="orangehrm-directory-card-body">
+      <span class="orangehrm-directory-card-icon">
+        <oxd-icon name="geo-alt-fill"></oxd-icon>
+      </span>
+      <span>
+        <div class="orangehrm-directory-card-subunit">
+          <oxd-text type="toast-message">
+            {{ employeeSubUnit }}
+          </oxd-text>
+        </div>
+        <div class="orangehrm-directory-card-location">
+          <oxd-text type="toast-message">
+            {{ employeeLocation }}
+          </oxd-text>
+        </div>
+      </span>
+    </div>
+    <div v-show="changeShowDetailsStatus && toggle">
+      <oxd-divider></oxd-divider>
+      <div class="orangehrm-directory-card-rounded-body">
```

Comment:
I think we discussed this detail section to be it's own component?

Reply:
Done